apps/req.c

```diff
@@ -32,6 +32,14 @@
 # include <openssl/dsa.h>
 #endif
 
+#ifdef __GNUC__
+# pragma GCC diagnostic ignored "-Wunused-function"
+#endif
+#ifdef __clang__
+# pragma clang diagnostic ignored "-Wunused-function"
+#endif
```

Comment:
Why is this needed here?

Reply:
Strangely, removing this does not cause errors for this one file.  fixup commit pushed.